tests/x11/yast2_snapper.pm

```diff
@@ -72,101 +31,26 @@ sub run() {
     # Start an xterm as root
     x11_start_program("xterm");
     assert_screen "xterm";
+    # Disable screen lock and blank screen for current Gnome session
+    script_run("gsettings set org.gnome.desktop.session idle-delay 0");
```

Comment:
On o3 i didn't see this module triggered by gnome or kde tests.
Add a check for this command.